packages/shared/src/components/profile/ActivitySection.tsx

```diff
@@ -2,18 +2,51 @@ import React, { Fragment, ReactElement, ReactNode } from 'react';
 import { InfiniteQueryObserverBaseResult } from 'react-query';
 import { Connection } from '../../graphql/common';
 import classed from '../../lib/classed';
+import { ClickableText } from '../buttons/ClickableText';
 
 export const ActivityContainer = classed('section', 'flex flex-col mt-10');
 
 export const ActivitySectionTitle = classed(
   'h2',
   'flex items-center mb-4 text-theme-label-primary font-bold typo-body',
 );
+
+export const ActivitySectionSubTitle = classed(
+  'span',
+  'flex flex-row mt-1 text-theme-label-tertiary typo-callout font-normal',
+);
+
 export const ActivitySectionTitleStat = classed(
   'span',
   'ml-1 text-theme-label-secondary font-normal',
 );
 
+interface TitleWithLinkProps {
+  title: string;
+  subtitle: string;
+  clickableTitle: string;
+  link: string;
+}
+
+export const TitleWithLink = ({
```

Comment:
The name is a bit misleading because when I see the word Title I expect to see also `h1`, `h2`, or any `hx` element but here it's a span. So not sure how we want to address it.

Reply:
Makes total sense.